Samples/PublicSamples/RecordingBot/docs/explanations/recording-bot-overview.md

```diff
@@ -0,0 +1,24 @@
+# High Level Overview
+
+![Image 1,](../images/Overview.svg)
+
+As seen in the image Teams clients communicate with the Teams Server / Teams Communication Platform. When a user under compliance recording policy initiates a call (a call with another user or a call into a meeting) in its teams client, the Teams platform gets the HTTPS notification URL from a bot service that is linked to the Compliance Recording Policy. The Teams Platform uses the notification URL to do a authorized http call to the Recording Application, informing about the user initiating a call, authorization is done with a JWT token in the headers.

```

Comment:
Clarity and capitalization. Avoid long sentences. Could be written as a sequence.